messages/it.json

```diff
@@ -1,63 +1,63 @@
-  "pickupPointsModal.cancelBackList": "Vedi tutti i punti di ritiro",
+  "pickupPointsModal.deliver": "Consegna",
+  "pickupPointsModal.pickup": "Ritiro",
+  "pickupPointsModal.items": "{itemsAmount, plural, =0 {} one {# oggetto} other {# oggetti}}",
+  "pickupPointsModal.cancelBackList": "Visualizza tutti i punti di ritiro",
   "pickupPointsModal.slaOption": "{name} - {price} - {shippingEstimate}",
-  "pickupPointsModal.free": "Gratuito",
+  "pickupPointsModal.free": "Gratis",
   "pickupPointsModal.selectPickupPoint": "Seleziona un punto di ritiro",
-  "pickupPointsModal.pointDetails": "Dettagli del punto di ritiro",
-  "pickupPointsModal.closeButton": "Chiude",
+  "pickupPointsModal.pointDetails": "Dettagli relativi al punto di ritiro",
+  "pickupPointsModal.closeButton": "chiudi",
   "pickupPointsModal.list": "Elenco",
   "pickupPointsModal.map": "Mappa",
-  "pickupPointsModal.pickupPoint.modify": "Modificare",
-  "pickupPointsModal.details": "Vedi i dettagli o cambia il punto di ritiro.",
-  "pickupPointsModal.searchLocationMap": "Cerca per via, codice postale (CAP), quartiere o città",
-  "pickupPointsModal.productsInPoint": "Disponibilità del prodotto in questo punto di ritiro",
+  "pickupPointsModal.pickupPoint.modify": "Modifica",
+  "pickupPointsModal.details": "Visualizza i dettagli o modifica il punto di ritiro.",
+  "pickupPointsModal.searchLocationMap": "Cerca una via, codice postale, quartiere o città",
+  "pickupPointsModal.productsInPoint": "Prodotto disponibile in questo punto di ritiro",
   "pickupPointsModal.aditionalInfo": "Altre informazioni",
   "pickupPointsModal.unavailable": "non disponibile",
-  "pickupPointsModal.unavailableItemsAmount": "{itemsAmount, plural, =0 {} one {# articolo non disponibile} altro {# articoli non disponibile}}",
-  "pickupPointsModal.allItemsAvailable": "Tutti i prodotti disponibili",
+  "pickupPointsModal.unavailableItemsAmount": "{itemsAmount, plural, =0 {} one {# oggetto non disponibile} other {# oggetto non disponibile}}",
+  "pickupPointsModal.allItemsAvailable": "Tutti i prodotti sono disponibili",
   "pickupPointsModal.noneItemsAvailable": "Nessun prodotto disponibile",
-  "pickupPointsModal.businessHours": "Orario commerciale",
-  "pickupPointsModal.confirmPoint": "Raccolta in questo punto di ritiro",
-  "pickupPointsModal.price": "{value, plural, =0 {Free} other {{formattedPrice}}}",
-  "pickupPointsModal.askGeolocation": "Cerca un indirizzo con la tua posizione",
-  "pickupPointsModal.marker": "Indicatore di geolocalizzazione",
-  "pickupPointsModal.geolocationDiscover": "Ti suggeriremo il punto di ritiro più vicino a te",
-  "pickupPointsModal.geolocationAsk": "Per questo, abbiamo bisogno della tua posizione",
+  "pickupPointsModal.businessHours": "Orari lavorativi",
+  "pickupPointsModal.confirmPoint": "Ritira presso questo punto di ritiro",
+  "pickupPointsModal.price": "{value, plural, one {} =0 {Gratis} other {{formattedPrice}}}",
```

Comment:
won't this `one {}` make the prices invisible for products with price equal to 1?

Reply:
Looks like there's an extra set of `{}` as well: `{{formattedPrice}}`

Reply:
Nope it is fine. 
Abou the `one={}` i'll remove it since it does not exist on the `en` and `pt` files (https://github.com/vtex/pickup-points-modal/blob/master/messages/pt.json#L25)

Reply:
Done.